app/model/editions/ImageOption.scala

```diff
@@ -0,0 +1,15 @@
+package model.editions
+
+import enumeratum.{EnumEntry, PlayEnum}
+import enumeratum.EnumEntry.Camelcase
+
+sealed abstract class ImageOption extends EnumEntry with Camelcase
+object ImageOption extends PlayEnum[ImageOption] {
+  case object UseArticleTrail extends ImageOption
+  case object Hide extends ImageOption
+  case object ReplaceCutout extends ImageOption
+  case object ReplaceSlideshow extends ImageOption
```

Comment:
I'm 99% sure that we're not supporting slideshows but @blishen should be able to clarify.

Reply:
I understand this came from @blishen so all good - just sounds a little like scope creep 😿